controllers/eventingauth_controller.go

```diff
@@ -37,7 +37,7 @@ const (
 	iasCredsSecretNamespace      string = "IAS_CREDS_SECRET_NAMESPACE"
 	iasCredsSecretName           string = "IAS_CREDS_SECRET_NAME"
 	defaultIasCredsNamespaceName string = "kcp-system"
-	defaultIasCredsSecretName    string = "eventing-auth-ias-creds"
+	DefaultIasCredsSecretName    string = "eventing-auth-ias-creds"
```

Comment:
Why this const is exported?

Reply:
it is used [[here](https://github.com/kyma-project/eventing-auth-manager/blob/70812009785a8cb7ffb16f82370ab74fa8634219/controllers/kyma_controller_test.go#LL117C17-L117C17)](url) and another place in this file